ext/date/php_date.stub.php

```diff
@@ -230,7 +230,7 @@ public function setISODate(int $year, int $week, int $dayOfWeek = 1) {}
     public function setTimestamp(int $timestamp) {}
 
     /**
-     * @return int|false
+     * @return int
```

Comment:
Now it returns just an "int", can it not be set as a return type on the function (line 236) itself?